dom-manipulation/main.js

```diff
@@ -0,0 +1,23 @@
+let clickNumber = 0;
+
+const $button = document.querySelector('.hot-button');
+
+const $click = document.querySelector('.click-count');
+
+$button.addEventListener('click', function (event) {
+  clickNumber++;
+  $click.innerHTML = 'Clicks:' + clickNumber;
```

Comment:
You want to avoid `innerHTML` because it's a security risk. It allows people to inject their own malicious scripts and content into your page. You should be using something like `textContent` instead